servers/rendering/shader_types.cpp

```diff
@@ -453,7 +453,7 @@ ShaderTypes::ShaderTypes() {
 	shader_modes[RS::SHADER_FOG].functions["fog"].built_ins["WORLD_POSITION"] = constt(ShaderLanguage::TYPE_VEC3);
 	shader_modes[RS::SHADER_FOG].functions["fog"].built_ins["OBJECT_POSITION"] = constt(ShaderLanguage::TYPE_VEC3);
 	shader_modes[RS::SHADER_FOG].functions["fog"].built_ins["UVW"] = constt(ShaderLanguage::TYPE_VEC3);
-	shader_modes[RS::SHADER_FOG].functions["fog"].built_ins["EXTENTS"] = constt(ShaderLanguage::TYPE_VEC3);
+	shader_modes[RS::SHADER_FOG].functions["fog"].built_ins["SIZE"] = constt(ShaderLanguage::TYPE_VEC3);
```

Comment:
If you are going to change the shader built in name, you also need to change the shader internals to rename here:
https://github.com/godotengine/godot/blob/9d3875dde75b95ea172c60188ca345162f2b38b9/servers/rendering/renderer_rd/renderer_scene_render_rd.cpp#L5382
Pass Size to the shader here:
https://github.com/godotengine/godot/blob/9d3875dde75b95ea172c60188ca345162f2b38b9/servers/rendering/renderer_rd/renderer_scene_render_rd.cpp#L4065-L4067
change the shader to use size internally:
https://github.com/godotengine/godot/blob/9d3875dde75b95ea172c60188ca345162f2b38b9/servers/rendering/renderer_rd/shaders/volumetric_fog.glsl#L40
and change the material to use Size as well:
https://github.com/godotengine/godot/blob/9d3875dde75b95ea172c60188ca345162f2b38b9/scene/resources/fog_material.cpp#L161

Reply:
Thanks, of course! I should have taken more care to check the shader-specific code after deciding that.

Reply:
@clayjohn In cases like fog_material:161, where the extents/size is used to determine something that seems fairly arbitrary (e.g. fog density based on volume), is it worth dividing by 2 to retain exactly the old values or should users just be required to double the density property? For now I'm leaving this one without dividing.

Reply:
Line 161 needs to be updated.  It scales the SDF based on it's distance to the edge of the volume and scales density accordingly.